app/controllers/verify/finance_controller.rb

```diff
@@ -37,6 +38,10 @@ def confirm_step_needed
       redirect_to verify_phone_path if idv_session.financials_confirmation == true
     end
 
+    def show_warning
+      flash[:warning] = t('idv.modal.finance.warning_html')
```

Comment:
maybe `flash.now` here instead? do you want it to persist beyond the current request?